content/academy/anti_scraping/proxies/using_proxies.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Using Proxies
+description: Learn how to use and automagically rotate proxies in your scrapers by using the Apify SDK.
+menuWeight: 1
+category: courses
+paths:
+- anti-scraping/proxies/using-proxies
+---
+
+# [](#using-proxies) Using Proxies
+
+In the [Web-Scraping for Beginners]({{@link web_scraping_for_beginners.md}}) course, we learned about the power of the Apify SDK, and how it can streamline the development process of web-crawlers. You've already seen how powerful the `apify` package is; however, what you've been exposed to thus far is only the tip of the iceberg.
```

Comment:
```suggestion
In the [Web-Scraping for Beginners]({{@link web_scraping_for_beginners.md}}) course, we learned about the power of the Apify SDK, and how it can streamline the development process of web crawlers. You've already seen how powerful the `apify` package is; however, what you've been exposed to thus far is only the tip of the iceberg.
```